### DISCUSSION TITLE
2022-07-27 Demo Day **SPECIAL EVENT**

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#258] [Weekly Meetings]
Weekly Meeting 66 is happening on Wednesday July 27th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- Check in on MSF. Let's check in on a plan for them to a. submit OMI to MSF, and b. how they will manage the delgate roles (checking email, passing along information, attending MSF meetings as OMI rep, etc.)
- Reminder: our next official demo day occurring during our Wednesday meeting on July 27th  

Kickoff with Evo and @mrmetaverse 16:00 UTC 
@RedEaglep1 9:05 UTC
@codefrau 9:15 UTC 
@fire  (and Saracen!) 16:25 UTC
@robertlong  16:35 UTC
@ JonYu 16:45 UTC
Wrap-up with Evo 16:55 UTC

#### Upcoming Wednesday Calendar `themes`: :calendar: (continued in the [omigroup/media repo](https://github.com/omigroup/media/discussions/43))
- Last week we discussed `Home space Interoperability`, Today we have a special full-session event where we will demo and discuss a variety of Metaverse interoperability topics. 

![OMI-demo-day-072722](https://user-images.githubusercontent.com/63426722/181118000-0e0d1ec1-3372-4cac-afcf-ec50ce6b833a.png)

#### Future Topics

- `content portability`  (August 3, 2022) 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Avatars` Pt II
- `Portals` Pt II 
- `Generative and AI tooling`

**Comment by madjin** [comment]
I got a short demo on animation interop

**Reply by mrmetaverse** [reply]
@madjin you think it could fall into `content portability`? :eyes:

**Comment by mrmetaverse** [comment]
# RedEagle of P1-OM 

- ![image](https://user-images.githubusercontent.com/63426722/181295841-23b99181-fbae-430f-a1b7-d8da77bc5fb0.png)
- ![image](https://user-images.githubusercontent.com/63426722/181295885-cfe9f964-0609-45d9-b1d9-abb812707ecd.png)
- ![image](https://user-images.githubusercontent.com/63426722/181296131-713eff87-febf-40e2-ae64-3115764b8a43.png)
- ![image](https://user-images.githubusercontent.com/63426722/181296289-830b24b6-1cf0-4763-b647-eaf6d6051e8e.png)
- the game is early, "Sky Jellies: Memories of Earth"
- teaching these cute characters about earth, is actually helping to train AI
- for example, if you teach this green jelly what a cucumber is, it will give you cucumber seeds. 
- ![image](https://user-images.githubusercontent.com/63426722/181296545-febfa3d9-0525-4b8f-b639-0114cc0b30c8.png)
- this is a different monetization model, gamers are rewarded for contributing, and game devs are rewarded with cash by empowering their gamers to fill out these annotations. 
- ![image](https://user-images.githubusercontent.com/63426722/181296920-495a15b8-b33e-4b3b-b8e8-e69976869375.png)
- ![image](https://user-images.githubusercontent.com/63426722/181297370-4fbcf23e-b193-4a39-888c-158157b429ef.png)
- 168 metaverse developers working together, interested in making the Metaverse free. If interested in joining reach out to @redeaglep1

**Comment by mrmetaverse** [comment]
# Vanessa @CodeFrau with Croquet! 
- Democratizing the Metaverse (like P1-OM) 
- ![image](https://user-images.githubusercontent.com/63426722/181298446-70b53745-683a-4050-9d90-8b72efbd80d4.png)
- ![image](https://user-images.githubusercontent.com/63426722/181298520-880314db-458e-40f3-acb3-d43ad22fa2b5.png)
- embracing open file types through JavaScript 
- ![image](https://user-images.githubusercontent.com/63426722/181298619-be86a87c-2c4b-4586-8574-8e0e29374a66.png)
- The OS they built Croquet on powers a collaboration framework. It is 3d-agnostic, and is a system that ensures that what we see is synchronized between different users. 
- ![image](https://user-images.githubusercontent.com/63426722/181298718-603a18e7-981c-4373-bf04-03a408ac45e8.png)
- recently won an Auggie award at AWE 
- this entire world is built client-side as a static web page. 
- developers have full autonomy, as it is all open source 
- ![image](https://user-images.githubusercontent.com/63426722/181298956-3dd713e0-6275-41dd-bc26-1024a53a5ec4.png)
- Croquet helps ensure interoperability between different Worlds 
- ![image](https://user-images.githubusercontent.com/63426722/181299078-5b450ea7-ed6d-4b57-942d-df3fa7413e21.png)
![image](https://user-images.githubusercontent.com/63426722/181299146-1dd51135-955b-4ecf-8991-1adb1f66762b.png)
- ![image](https://user-images.githubusercontent.com/63426722/181299186-60b71ffb-3949-4a9d-a83a-f79784635a86.png)
- ![image](https://user-images.githubusercontent.com/63426722/181299217-9dea1d85-7bb3-48c5-a486-5978ae14ffcb.png)
- and Jenn can come through the portal too! 
- ![image](https://user-images.githubusercontent.com/63426722/181299259-b04079bb-a302-4665-8bae-a7f4452b8b69.png)
- Synchronizing cameras across Worlds 
- ![image](https://user-images.githubusercontent.com/63426722/181299447-050ce844-84a1-4cbc-b848-abb98c73ed45.png)
- ![image](https://user-images.githubusercontent.com/63426722/181299521-7666abf3-d182-4051-adc1-ce4a09cad46b.png)
- With the avatar able to stay the same! 
- The way Croquet does this: Locally on the machine avatar information is sent, including the source URL for the glb. This remains decentralized without required registration. 
- The metashell helps coordinate worlds through local post-messages 
- ![image](https://user-images.githubusercontent.com/63426722/181299799-b3ecb17f-9147-43cd-b394-2d7e58735f0e.png)
- Doesn't have to be a croquet World on the other side. Simply need to leverage the meta-shell protocol. It has limited functionality: Can open a portal, and send the camera to a lower iframe.
- the platform is the web. You don't have to conform in order to be interoperable. That is a very compelling value proposition.

**Reply by codefrau** [reply]
Amazing summary, thank you! 💜
One correction: Our Jenn has two "n" (you couldn't guess from her handle, that's okay)

**Comment by mrmetaverse** [comment]
# Fire and V-Sekai
- ![image](https://user-images.githubusercontent.com/63426722/181301015-603eafcc-1f2e-4791-bac9-4cfca48b791f.png)
- Godot editor using a scene from Mozilla hubs 
- ![image](https://user-images.githubusercontent.com/63426722/181301062-84973bf0-c550-4d14-843e-20e5973c1c2b.png)
- Have been working on gLTF extensions with OMI. The idea of this demo is that they recently achieved putting gLTF extensions into the godot editor. 
- Can actually add what they want using the importer extension, and can wrtite in their own scripting language 
- 
![image](https://user-images.githubusercontent.com/63426722/181301289-d60b2d43-592a-4686-b6ae-74b0cf436c77.png)
- devs who are using V-Sekai can use interoperable extensions written by various groups from OMI's audio extensions, Hubs scenes, VRM avatars, and many many more! 
- ![image](https://user-images.githubusercontent.com/63426722/181301392-54148c37-e370-4135-88d3-8dba0ec567b0.png)
- Also working on implementing the VRM importer to Godot. Has been finished for a while but rarely demoed. 
- VRM is an interoperable avatar standard that can be downloaded as a .VRM file and imported into virtual Worlds and metaverse platforms. 
- ![image](https://user-images.githubusercontent.com/63426722/181301550-6eefefc4-ff3a-427e-abb5-52baa26f3202.png)
- In Godot engine now you could import the VRM, and assist with the creative workflow from inside of Godot engine. 
- ![image](https://user-images.githubusercontent.com/63426722/181301666-30e95735-93fc-4981-8e18-791e00625027.png)
- Plans to make this an exporter as well to use in Blender, Unity, and many others
- Jin shared a resource for [VRM Samples](https://github.com/madjin/vrm-samples) in chat. 
- Saracen joined in on the stream 
- Godot is a flexible, open-source game engine 
- The team is working on tooling for Godot 
- ![image](https://user-images.githubusercontent.com/63426722/181301965-d0d7d541-f9cc-42e6-badc-e105d392283f.png)
- they have a plugin that allows for a one-button-click deployment 
- ![image](https://user-images.githubusercontent.com/63426722/181302053-e0f4d5b0-ffa6-4777-9599-936dbe93f1d1.png)
- as well as an experimental back-end with elixer 
- can upload "my map" 
- ![image](https://user-images.githubusercontent.com/63426722/181302203-5fe32951-85b0-4dae-a8b4-64feb7dd43dc.png)
- ![image](https://user-images.githubusercontent.com/63426722/181302248-d2ec8fee-cd6e-4a04-9f4c-53b1065fe78f.png)
- World loaded! 
- ![image](https://user-images.githubusercontent.com/63426722/181302297-77a4d7e1-47c7-417a-b6c8-88bde3c0e9b7.png)
- also wrote a tool that allows for import from raw Unity packages 
- ![image](https://user-images.githubusercontent.com/63426722/181302443-5584b526-c755-495b-a0d3-c9590151e8dc.png)
- so here is a famous unity demo, we can implement multiplayer and more! 
- ![image](https://user-images.githubusercontent.com/63426722/181302627-a193e367-da0f-4ce6-9abc-7c285374fa89.png)

**Comment by mrmetaverse** [comment]
# Robert Long (co-chair, OMI) demoing Third Room
- ![image](https://user-images.githubusercontent.com/63426722/181302953-88776fb0-d513-4589-91c0-3851aea1824c.png)
- In [our last OMI demo night](https://www.youtube.com/watch?v=e26UJRCGfGk&t=23s&ab_channel=OpenMetaverseInteroperabilityGroup), Robert referred to this as "a side project," and a lot has changed since then. 
- ![image](https://user-images.githubusercontent.com/63426722/181303192-30111245-f362-402e-84d9-59aab12346d0.png)
- ![image](https://user-images.githubusercontent.com/63426722/181303466-a38af1a1-52e0-4b2a-94d7-b58c073c194d.png)
- Matrix has a ton of features around rooms and messaging and we get to leverage all of them 
- Live Room! 
- ![image](https://user-images.githubusercontent.com/63426722/181303814-e18f2780-1fd9-4606-8a89-e97eeec69513.png)
- ![image](https://user-images.githubusercontent.com/63426722/181303851-b352d99a-693e-423a-bd7a-3c1b947dbdd7.png)
- early multiplayer is working (and physics!) 
- ![image](https://user-images.githubusercontent.com/63426722/181303919-29ca5c7e-2bfe-42d9-8194-0ea1d4e60e79.png)
- in-world chat
- ![image](https://user-images.githubusercontent.com/63426722/181303960-4eddd3c2-e791-45aa-b04b-699b1c2cc969.png)
- jump to new worlds
- ![image](https://user-images.githubusercontent.com/63426722/181304001-3e69dc0d-d08e-4172-ac44-a283e43eeca3.png)
- ![image](https://user-images.githubusercontent.com/63426722/181304069-fbb58174-5fa9-4b85-9e64-11cd4f6e7cb8.png)
- load in scenes from Mozilla Spoke 
- ![image](https://user-images.githubusercontent.com/63426722/181304140-66feb14e-908a-41f2-b30e-cee7293059ef.png)
- ![image](https://user-images.githubusercontent.com/63426722/181304195-2760faa2-23c0-4934-b8da-eceaf4606207.png)
- everything just works, and thats the power of interoperability with gLTF. 
- one last scene, with photogrammetry 
- ![image](https://user-images.githubusercontent.com/63426722/181304285-382e5f96-3061-4322-a1a2-3d1a4d6be9e0.png)
- ![image](https://user-images.githubusercontent.com/63426722/181304318-d901b015-76e2-4a76-9a52-fa7109df8ca4.png)
- even works well on an ultrawide 120hz gaming monitor. 
- ![image](https://user-images.githubusercontent.com/63426722/181304429-ae865122-0906-437c-8d8b-9f2cbf93ed06.png)
- supports firefox nightly (but not regular firefox yet) 
- Safari working through some CORS issues 
- looking to support Avatar standards group soon and the hubs avatar spec 
- (not shown) in-World you can press the `~` key and make edits in-world and publish directly to Matrix as your home server. 
- The thirdroom application is built on an open source system called "Manifold". 
- ![image](https://user-images.githubusercontent.com/63426722/181305372-75f61294-e06c-45ad-92d9-3b01dbe5c715.png)

**Comment by robertlong** [comment]
Here are the links/slides from my talk:

Third Room Alpha: https://thirdroom.io/ (See Github below for instructions on how to join)
GitHub: https://github.com/matrix-org/thirdroom
Matrix Chat Room: https://matrix.to/#/%23thirdroom-dev:matrix.org
Slides: [ThirdRoomOMIDemoDay2.pdf](https://github.com/omigroup/omigroup/files/9201224/ThirdRoomOMIDemoDay2.pdf)

**Comment by mrmetaverse** [comment]
# Jon Yu of Mythologi.es
- no code platform, geo-located and persistent platform 
- allow people to tell stories and create experiences 
- ![image](https://user-images.githubusercontent.com/63426722/181306900-41786f45-d8e2-48ed-8894-50b615829131.png)
- want to be able to plug into different platforms including VR 
- Projects are essentially Worlds 
- ![image](https://user-images.githubusercontent.com/63426722/181306949-c1cee991-0025-446b-80de-9f689b5160c7.png)
- ![image](https://user-images.githubusercontent.com/63426722/181306985-ffceb8a5-b849-4e0a-8e1a-e47a1ef802f0.png)
- content management in place that supports gLTF, etc. 
- ![image](https://user-images.githubusercontent.com/63426722/181307040-a2c54806-4ac4-4852-b7bb-801fa5c43762.png)
- can download objects to your device (this was demoed on ipad!) 
- ![image](https://user-images.githubusercontent.com/63426722/181307101-493fc528-1ae2-4048-a434-8eebb6dd29e6.png)
- ![image](https://user-images.githubusercontent.com/63426722/181307119-838cadb3-bd62-4f73-be75-c6a087b00e43.png)
- ![image](https://user-images.githubusercontent.com/63426722/181307178-7ab52413-453a-412b-a2bb-f131461cbc3b.png)
- every user has their own library, and can engage as collaborators 
- ![image](https://user-images.githubusercontent.com/63426722/181307230-ab1d2751-611d-4d6e-a932-647719384154.png)
- creating the colleseum 
- ![image](https://user-images.githubusercontent.com/63426722/181307274-df08800d-54a2-42b3-aab7-4cbad4044244.png)
- define geo-fence 
- ![image](https://user-images.githubusercontent.com/63426722/181307311-50a77257-5b47-45c1-b8c6-ee40a85b042d.png)
- created an area around a location 
- ![image](https://user-images.githubusercontent.com/63426722/181307359-6e863ff0-db96-4310-bfc4-9a3f1dc94c41.png)
- ![image](https://user-images.githubusercontent.com/63426722/181307438-5c2c69b3-5570-4c39-8a99-a6f86cf01523.png)
- ![image](https://user-images.githubusercontent.com/63426722/181307398-e47d1290-9752-4c57-ba54-8f659bf01b24.png)
- ![image](https://user-images.githubusercontent.com/63426722/181307493-6e2053de-5ae7-489a-8ed8-3e42e6b013a2.png)
- define your anchor and marker 
- ![image](https://user-images.githubusercontent.com/63426722/181307528-90251d64-caf3-465d-9921-a261e0b934ad.png)
- add spaceman 
- ![image](https://user-images.githubusercontent.com/63426722/181307574-2dd0ea78-ba34-4ddf-ad08-8c6663eb5d69.png)
- ![image](https://user-images.githubusercontent.com/63426722/181307610-df517faa-3d8a-41f4-a709-0163c3e1a7b5.png)
- line space person up 
- ![image](https://user-images.githubusercontent.com/63426722/181307653-bc3800dc-3330-4742-8e73-24900fa6e5a9.png)
- playful and easy, auto-syncs with cloud for real-time collaboration 
- ![image](https://user-images.githubusercontent.com/63426722/181307716-c4cdc281-c95e-4a9f-88f9-777c50cfa401.png)
- ![image](https://user-images.githubusercontent.com/63426722/181307788-6d7cf28f-9cac-41d6-a210-46c5d3d45646.png)
- you can kinda drop worlds inside of worlds 
- ![image](https://user-images.githubusercontent.com/63426722/181307864-9b07adab-99c3-4771-9e5c-01a9fa020018.png)
- supports photogrammetry 
- ![image](https://user-images.githubusercontent.com/63426722/181307956-adba60c1-1dc0-4a4e-957e-9d50066ebc1e.png)
- audit real world dimensions for market placement 
- ![image](https://user-images.githubusercontent.com/63426722/181307988-e188b54e-def5-4175-8fe4-60bcc65aaf66.png)
- As a player plays through these experiences, the position they are in dictates what is being revealed to them. 
- ![image](https://user-images.githubusercontent.com/63426722/181308094-2f2205b7-b0b8-4312-9d25-2456e2fb1f8e.png)
- Amp asset even plays audio!
- Building their own engine called "Geist" 
- you can sequence peoples experience by having assets reveal themselves based on stepping inside of certain nodes 
- ![image](https://user-images.githubusercontent.com/63426722/181308552-13a440ff-f2f2-4fbb-bf34-ce4660c2a67a.png)
- ![image](https://user-images.githubusercontent.com/63426722/181308594-64d73dbc-5f45-4a1a-872f-8606eddae8fc.png)
- can create relationships between items across the scene 
- ![image](https://user-images.githubusercontent.com/63426722/181308654-ccba46b2-652c-4a0f-9ecd-3249a82e950d.png)
